modules/claims_api/lib/library_base.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'claims_api/claim_logger'
+require 'custom_error'
+
+module ClaimsApi
+  class LibraryBase
+    def error_handler(error, async = true) # rubocop:disable Style/OptionalBooleanParameter
+      ClaimsApi::CustomError.new(error, async).build_error
+    end
+
+    def log_outcome_for_claims_api(action, status, response)
+      ClaimsApi::Logger.log(action,
+                            detail: "#{action} #{status}: #{response}")
+    end
+
+    def handle_transaction(service)
+      resp = client.get(service).body[:items]
+      log_outcome_for_claims_api(service, 'success', resp)
```

Comment:
This line adds new functionality (logging successful requests). Do we want these logs?

Reply:
@tycol7 Sometimes it is nice to see which calls where successful, especially when the service gets called multiple times for one endpoint. Debatable, for sure.

Reply:
I agree here that this extra step is probably not needed. additionally, some of these responses from BRD can be massive as well, not sure we want to log all of that on every request - /disabilities in staging has over 4,000 lines. On top of that, we make multiple calls to BRD at the current time, so you're talking 3'ish times the expected volume of 526 submissions (roughly 4,000+ a day)... that's 12,000+ logs of their response in it's current state.

As straightforward as possible, I'd suggest in rescuing a `Net::HTTPGatewayTimeout` in the service as mentioned in the ticket, and then raise a `Lighthouse::Timeout`(assuming that template is aligning with [LH standards](https://department-of-veterans-affairs.github.io/va-api-standards/errors/)). 
Do we know how much that differs from how the `::Common::Exceptions::GatewayTimeout`? Think I would still lean a new one as it's best if we can control / change the error - plus we'll be able to easily extract just the LH errors when we move to LHDI.